src/pages/GalleryDetails/components/GalleryEditor.tsx

```diff
@@ -0,0 +1,123 @@
+import EditRoundedIcon from '@mui/icons-material/EditRounded';
+import { Box, IconButton } from '@mui/material';
+import { makeStyles } from 'makeStyles';
+import { useCallback, useEffect, useState } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+import { Gallery, GalleryRequired } from 'types';
+
+import { useAlbumsById, useDeleteAlbum, useUpdateAlbum } from 'hooks/Gallery';
+import { useSnackbar } from 'hooks/Snackbar';
+
+import SubmitButton from 'components/inputs/SubmitButton';
+import TextField from 'components/inputs/TextField';
+import Dialog from 'components/layout/Dialog';
+import VerifyDialog from 'components/layout/VerifyDialog';
+
+const useStyles = makeStyles()((theme) => ({
+  margin: {
+    margin: theme.spacing(2, 0, 1),
+    borderRadius: theme.shape.borderRadius,
+    overflow: 'hidden',
+  },
+}));
+
+type GalleryEditorProps = {
+  slug: string;
+};
+
+type FormValues = Omit<Gallery, 'id' | 'created_at' | 'updated_at'>;
+
+const GalleryEditor = ({ slug }: GalleryEditorProps) => {
+  const { classes } = useStyles();
+  const { data } = useAlbumsById(slug);
+  const editGallery = useUpdateAlbum(slug);
+  const deleteGallery = useDeleteAlbum(slug);
+  const showSnackbar = useSnackbar();
+  const { handleSubmit, register, formState, reset } = useForm<FormValues>();
+  const setValues = useCallback(
+    (newValues: Gallery | null) => {
+      reset({
+        title: newValues?.title || '',
+        description: newValues?.description || '',
+        image_alt: newValues?.image_alt || '',
+      });
+    },
+    [reset],
+  );
+
+  useEffect(() => {
+    setValues(data || null);
+  }, [data, setValues]);
+
+  const remove = async () => {
+    deleteGallery.mutate(null, {
+      onSuccess: () => {
+        showSnackbar('Slettet', 'success');
+      },
```

Comment:
Send bruker til galleri-forsiden ved sletting av galleri
```suggestion
      onSuccess: () => {
        showSnackbar('Albumet ble slettet', 'success');
        navigate(URLS.galleries)
      },
```